### PR TITLE
Report :id of Error

### LINKED ISSUE #13: Display the rule after an error in flycheck
When I see a `vale` error the name of the rule that was violated is not shown. This would be useful for when I want to disable the rule. I have to run vale from the command line to get the name of the rule. Other flycheck packages, such as shellcheck, show the rule in parentheses.

Example:

```
 scratch…   170 176 warning         'all of' is too wordy (vale)
```

### PR DESCRIPTION
**Pull request by juergenhoetzel**
Fixes #13.

### CODE REVIEW

**Comment by abingham** [comment]
Thanks again! Are you interested in taking over this project? I don't use emacs any more, but I'd love to see this maintained.

**Comment by juergenhoetzel** [comment]
> Thanks again! Are you interested in taking over this project? I don't use emacs any more, but I'd love to see this maintained.

You are welcome to add me as a collaborator.
